CommandHelper/LocalPackages/commands/chars/wildmutt.yml

```diff
@@ -0,0 +1,37 @@
+buffs:
```

Comment:
buffs have been split into 2 sections now, effects and items. please see superman.yml for reference